scenario_player/main.py

```diff
@@ -89,6 +91,22 @@ def load_account_obj(keystore_file, password):
         return account
 
 
+def get_password(password, password_file):
+    if password_file:
+        password = open(password_file, "r").read().strip()
+    if password == password_file is None:
+        password = click.prompt(text="Please enter your password: ", hide_input=True)
+    return password
+
+
+def get_account(keystore_file, password):
```

Comment:
Looking at this I just realized this could be moved into `load_account_obj` instead of adding a func that, essentially, only adds a try/except block.

```
def load_account_obj(keystore_file, password):
    with open(keystore_file, "r") as keystore:
        try:
            account = Account(json.load(keystore), password, keystore_file)
        except ValueError:
            raise WrongPassword
        log.info("Using account", account=to_checksum_address(account.address))
        return account
```